conseil-common/src/main/scala/tech/cryptonomic/conseil/common/bitcoin/rpc/BitcoinRpcClient.scala

```diff
@@ -0,0 +1,98 @@
+package tech.cryptonomic.conseil.common.bitcoin.rpc
+
+import cats.effect.{Concurrent, Resource}
+import com.typesafe.scalalogging.LazyLogging
+import fs2.{Pipe, Stream}
+import io.circe.generic.auto._
+import org.http4s.circe.CirceEntityDecoder._
+import org.http4s.circe.CirceEntityEncoder._
+
+import tech.cryptonomic.conseil.common.rpc.RpcClient
+import tech.cryptonomic.conseil.common.bitcoin.rpc.BitcoinRpcCommands._
+import tech.cryptonomic.conseil.common.bitcoin.rpc.json.{Block, BlockchainInfo, Transaction}
+
+/**
+  * Bitcoin JSON-RPC client according to the specification at https://developer.bitcoin.org/reference/rpc/
+  *
```

Comment:
I think that we could improve by describing how the api is supposed to be used, explaining shortly how to handle a Stream, and more so how Pipes works and are meant to be used.